client/my-sites/sidebar/sidebar.jsx

```diff
@@ -336,6 +336,11 @@ export class MySitesSidebar extends Component {
 		);
 	}
 
+	trackStoreClick = () => {
+		analytics.tracks.recordEvent( 'calypso_woocommerce_store_nav_item_click' );
```

Comment:
I'll make sure each track event in MC gets a description so we will know what these mean when viewing them.